polaris-migrator/src/migrations/replace-sass-z-index/replace-sass-z-index.ts

```diff
@@ -0,0 +1,119 @@
+import type {API, FileInfo, Options} from 'jscodeshift';
+import postcss, {Plugin} from 'postcss';
+import valueParser, {FunctionNode, Node} from 'postcss-value-parser';
+
+import {POLARIS_MIGRATOR_COMMENT} from '../../constants';
+
+interface PluginOptions extends Options {
+  namespace?: string;
+}
+
+const processed = Symbol('processed');
+
+function isNumericOperator(node: Node): boolean {
+  return (
+    node.value === '+' ||
+    node.value === '-' ||
+    node.value === '*' ||
+    node.value === '/' ||
+    node.value === '%'
+  );
+}
+
+const zIndexMap = {
+  content: '--p-z-1',
+  overlay: '--p-z-2',
+};
+
+const isValidElement = (
+  element: unknown,
+): element is keyof typeof zIndexMap => {
+  return Object.keys(zIndexMap).includes(element as string);
+};
+
+const hasMoreThanOneArgument = (node: FunctionNode) => {
+  if (!node.nodes || !Array.isArray(node.nodes)) return false;
+  return node.nodes && node.nodes.length > 1;
+};
```

Comment:
```suggestion
const hasMoreThanOneArgument = (node: FunctionNode) => node.nodes.length > 1;
```

Function nodes [should always have a `nodes` array](https://github.com/TrySound/postcss-value-parser/blob/11a4436e74930f4eaf9cbdb2063656bcce8ebf6a/lib/index.d.ts#L52). Did you run into an edge case?

Reply:
Good catch, was previous dealing with bad typing; no reason to have this check now 👍